packages/@aws-cdk/pipelines/lib/codepipeline/codepipeline-source.ts

```diff
@@ -51,16 +52,37 @@ export abstract class CodePipelineSource extends Step implements ICodePipelineAc
    * Example:
    *
    * ```ts
-   * const bucket: IBucket = ...
-   * CodePipelineSource.s3(bucket, {
-   *   key: 'path/to/file.zip',
+   * import * as ecr from '@aws-cdk/aws-ecr';
```

Comment:
@alexpulver  : should the following comments have been modified ? I think they still apply to the `s3` factory method.